src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -1,38 +1,27 @@
 import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
 import Spinner from "lib/Components/Spinner"
 import { isCloseToBottom } from "lib/utils/isCloseToBottom"
-import { get, once } from "lodash"
+import { once } from "lodash"
 import PropTypes from "prop-types"
 import React, { Component } from "react"
 import { FlatList, ScrollView, Text, View } from "react-native"
-import { RelayPaginationProp } from "react-relay"
+import { createPaginationContainer, graphql, RelayPaginationProp } from "react-relay"
 import styled from "styled-components/native"
 import { SectionHeader } from "./SectionHeader"
 
 const DEFAULT_TITLE = "Lots by Artists You Follow"
-export const PAGE_SIZE = 10
-
-const Container = styled.View`padding: 10px;`
-
-interface Props {
-  relay?: RelayPaginationProp
-  title?: string
-  saleArtworks?: {
-    edges: Array<{
-      node: object | null
-    }> | null
-  }
-}
+const PAGE_SIZE = 10
 
 export function LotsByFollowedArtists(props: Props) {
```

Comment:
What’s the inferred type of this? I wonder what happens if you would type it as a stateless component:

```js
const LotsByFollowedArtists: React.SFC<Props> = props => { … }
```

Reply:
Looks like it comes back as a `JSX.Element`:

<img width="543" alt="screen shot 2017-12-05 at 12 05 54 pm" src="https://user-images.githubusercontent.com/236943/33628182-c14d65c8-d9b4-11e7-9ac8-9efcee0efd8b.png">

But typing it to `React.SFC` returns more valuable info:

<img width="690" alt="screen shot 2017-12-05 at 12 08 01 pm" src="https://user-images.githubusercontent.com/236943/33628265-035613ca-d9b5-11e7-9e39-c9a3f1a698ad.png">

Will update 👍

Reply:
👍